src/cmd/cli/command/compose.go

```diff
@@ -15,9 +16,34 @@ import (
 	"github.com/DefangLabs/defang/src/pkg/types"
 	defangv1 "github.com/DefangLabs/defang/src/protos/io/defang/v1"
 	"github.com/bufbuild/connect-go"
+	compose "github.com/compose-spec/compose-go/v2/types"
 	"github.com/spf13/cobra"
 )
 
+var MANAGED_RESOURCES = []string{"x-defang-redis", "x-defang-postgres"}
+
+func isOnlyManagedResources(project *compose.Project) bool {
```

Comment:
Good function to write unit tests for:

- no services
- one managed service
- one regular service
- one of each
- etc